src/js/angular/getImgService.test.js

```diff
@@ -0,0 +1,134 @@
+describe('getImagesService', function(){
+    'use strict';
+
+    var $http;
+    var ImgService;
+
+    beforeEach(module('flickrPOC'));
+
+    beforeEach(inject(function(_$http_, _images_){
+        $http = _$http_;
+        ImgService = _images_;
+    }));
+
+    describe('loadPics', function(){
+        describe('stub service call', function(){
+            it('should exist', function(){
+                expect(ImgService.loadPics).to.exist();
+            });
+            it('should use $http to retrieve data', function(){
+                var spy = sinon.spy($http, 'get');
+                var method = "flickr.photos.search";
+                var api_key = '74e47a159e15cbcb6139ba9c9df64c13';
+                var user_id = "138698049@N03";
+                var mockUrl = 'https://api.flickr.com/services/rest?method=' + method + '&api_key='+ api_key + '&user_id=' + user_id + '&format=json&nojsoncallback=1';
+
+                ImgService.loadPics();
+                expect(spy).to.have.been.calledWith(mockUrl);
+            });
+
+            it('should return a promise', function(){
+               expect(ImgService.loadPics().then).to.exist();
+            });
```

Comment:
I'd suggest moving this `it()` block to in between the other 2. As I read tests, I view them as chapters in a book, *The Story of the Image Service* in this case. :smile: 

Certainly more of a preference than a standard, but want to pass along all my quirks. 

So, if that's the case, I'd like to ensure 3 things happen in the order of simple to complex:
1. The loadPics method exists
2. The loadPics method returns a promise
3. The loadPics method uses a specific service ($http) with the proper URL